lib/browser/api/browser-window.js

```diff
@@ -162,6 +162,9 @@ Object.assign(BrowserWindow.prototype, {
   inspectServiceWorker () {
     return this.webContents.inspectServiceWorker()
   },
+  showCertificate () {
```

Comment:
is this needed for anything? There will never be a certificate for the browser-window